### ISSUE TITLE
Issue with Storage Management SDK in Local Emulator (under Full IIS)

### ISSUE THREAD

**Issue by npickett** [Azure/azure-sdk-for-net#897]
Hi All --

When using the Storage Management SDK with an Azure Web Role using IIS Web Server and Full Emulator, I get the following error:

```
<Error xmlns="http://schemas.microsoft.com/windowsazure" xmlns:i="http://www.w3.org/2001/XMLSchema-instance"><Code>ForbiddenError</Code><Message>The server failed to authenticate the request. Verify that the certificate is valid and is associated with this subscription.</Message></Error>
```

I am using VS 2013 with the Azure 2.3 SDK and downloaded the latest code for the Azure Storage Management SDK from here: 

https://github.com/Azure/azure-sdk-for-net/tree/master/src/StorageManagement

If I change to use IIS Express, request is authenticated and desired result is returned.  I can provide solution (VS2013) for replicating this issue.

Does anyone know why this happens and if there is a workaround?  

Thank you. -Nate

**Comment by ivanfioravanti** [comment]
We are experiencing exactly the same error. Any news on this?

**Comment by NTTLuke** [comment]
We have the same error with Scheduler Library with the last Management Library Update.

**Comment by markcowl** [comment]
@npickett Can you please elaborate on the scenario?  The storage management SDK is only used for creating and managing storage ACCOUNTS.  There is a single, logical account in the emulator, so there is no scenario for using the storage management sdk with the emulator.  The storage sdk, which you use to manage tables, blobs and queues within a single storage account is not shipped from this repository.  If this question is about the storage sdk, please report this issue in the appropriate repository:  https://github.com/Azure/azure-storage-net.. If not, please elaborate on your storage management sdk scenario.

**Comment by npickett** [comment]
Hi Mark,

I have posted a test project that will replicate the issue we have been experiencing.  

https://github.com/npickett/azure-sdk-for-net-storage-mgmt-repo

I experienced issues when using the azure run-time emulator locally on my machine pointed to external azure storage endpoints and when "Use IIS Web Server" is selected.  Below is a screenshot of the configuration setup that breaks and returns a "403 Forbidden" error after trying to authenticate a storage management sdk call:

![sdkdebug](https://cloud.githubusercontent.com/assets/1013081/6508262/c5ecbb04-c325-11e4-996e-806791c952e6.PNG)

When "Use IIS Express" is selected, it works.  I assume it must be something to do with how permissions are set up under IIS vs. IIS Express for retrieving/finding the management certificate.

Thank you. -Nate
